docs/configuration.md

```diff
@@ -635,7 +635,10 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.shuffle.maxChunksBeingTransferred</code></td>
   <td>Long.MAX_VALUE</td>
   <td>
-    The max number of chunks allowed to being transferred at the same time on shuffle service.
+    The max number of chunks allowed to be transferred at the same time on shuffle service.
+    Note that new incoming connections will be closed when the max number is hit. The client will
+    retry according to the shuffle retry configs (see spark.shuffle.io.maxRetries and
+    spark.shuffle.io.retryWait), if those limits are reached the task will fail with fetch failure.
```

Comment:
thanks for updating, sorry I forgot to ask you to put spark.shuffle.io.maxRetries and spark.shuffle.io.retryWait  inside <code></code> so they show up as references like in other places.  See the description of 
spark.shuffle.compress
 as an example on how ti references other configs.